main/resources/com/google/refine/grel/ControlEvalError.properties

```diff
@@ -13,3 +13,4 @@ expects_second_arg_var_name={0} expects second argument to be a variable name
 expects_second_arg_index_var_name={0} expects second argument to be the index's variable name
 expects_third_arg_element_var_name={0} expects second argument to be the element's variable name
 expects_second_third_args_different={0} expects the second and third arguments to be different variable names
+expects_array_arg={0} is not an array
```

Comment:
I wouldn't introduce a new translation string: why not keep the existing one? It does not seem to be used anywhere so you can modify it as you wish.
Looking at the pattern of the error messages around it, I would suggest something like `{0} expects an array as its first argument`, where `{0}` gets replaced by the function name, not the value.

Reply:
@wetneb That was my initial thought too. However I think it might be the case that `filter` actually accepts the same types for the first argument as `forEach` and `forEachIndex`. If this could be verified, then having a new translation string like proposed here and using it for all cases could make sense?